space-framework/pkg/msclientlib/client.go

```diff
@@ -42,14 +42,15 @@ type KubestellarSpaceInterface interface {
 	//Space(name string, namespace ...string) (client mcclientset.Interface)
 
 	// ConfigForSpace returns rest config for given space.
-	ConfigForSpace(name string, namespace ...string) (*rest.Config, error)
+	ConfigForSpace(name string, providerNS string) (*rest.Config, error)
 }
 
 type multiSpaceClient struct {
 	ctx context.Context
 	// Currently configs holds the kubeconfig as a string for each Space.
 	// This is temporary as Space is bing changed to use secrets instead of holding the config as
 	// text string
+	// The key to the map is the spaceName+ProviderNS  (currently we use the provider NS and not directly the provider name)
```

Comment:
"spaceName+ProviderNS" reads like code --- _different_ code than I see in `namespaceKey`